### PR TITLE
[4.0] media action fields and a non correct header

### PR DESCRIPTION
**Pull request by zero-24**
Pull Request für Issue #807 

### Zusammenfassung der Änderungen

 plg_media-action_crop.ini
 plg_media-action_crop.sys.ini
 plg_media-action_resize.ini
 plg_media-action_resize.sys.ini
 plg_media-action_rotate.ini
 plg_media-action_rotate.sys.ini

This also is fixing an issue with the header of the namespace map files.

### CODE REVIEW

File: administrator/language/de-DE/plg_media-action_crop.ini [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+; License GNU General Public License version 2 or later; see LICENSE.txt
+; Note : All ini files need to be saved as UTF-8
+
+PLG_MEDIA-ACTION_CROP="Medienfunktion - Zuschneiden"
+PLG_MEDIA-ACTION_CROP_LABEL="Zuschneiden"
+PLG_MEDIA-ACTION_CROP_PARAM_DEFAULT_RATIO="Standard-Seitenverhältnis"
+PLG_MEDIA-ACTION_CROP_PARAM_HEIGHT="Höhe"
+PLG_MEDIA-ACTION_CROP_PARAM_LANDSCAPE="Landschaft"
+PLG_MEDIA-ACTION_CROP_PARAM_NO_RATIO="Keine"
+PLG_MEDIA-ACTION_CROP_PARAM_PORTRAIT="Porträt"
```
Comment (tecpromotion) [inline administrator/language/de-DE/plg_media-action_crop.ini]:
```suggestion
PLG_MEDIA-ACTION_CROP_PARAM_PORTRAIT="Portrait"
```

File: administrator/language/de-DE/plg_media-action_crop.ini [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+PLG_MEDIA-ACTION_CROP_PARAM_WIDTH="Breite"
+PLG_MEDIA-ACTION_CROP_PARAM_X="X-Axe"
```
Comment (tecpromotion) [inline administrator/language/de-DE/plg_media-action_crop.ini]:
```suggestion
PLG_MEDIA-ACTION_CROP_PARAM_X="x-Achse"
```

File: administrator/language/de-DE/plg_media-action_crop.ini [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+PLG_MEDIA-ACTION_CROP_PARAM_Y="Y-Axe"
```
Comment (tecpromotion) [inline administrator/language/de-DE/plg_media-action_crop.ini]:
```suggestion
PLG_MEDIA-ACTION_CROP_PARAM_Y="y-Achse"
```

File: administrator/language/de-DE/plg_media-action_resize.ini [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+PLG_MEDIA-ACTION_RESIZE_BATCH_LABEL="Batch-Einstellungen"
+PLG_MEDIA-ACTION_RESIZE_BATCH_MAX_HEIGHT_LABEL="Maximale Höhe des Bildes"
+PLG_MEDIA-ACTION_RESIZE_BATCH_MAX_HEIGHT_DESC="Die maximale Höhe eines Bildes, dessen Größe geändert werden soll. Wenn dieses Feld leer ist, wird keine Größenänderung durchgeführt."
+PLG_MEDIA-ACTION_RESIZE_BATCH_MAX_WIDTH_LABEL="Maximale Breite des Bildes"
+PLG_MEDIA-ACTION_RESIZE_BATCH_MAX_WIDTH_DESC="Die maximale Breite eines Bildes, die in der Größe geändert werden soll. Wenn dieses Feld leer ist, wird keine Größenänderung durchgeführt."
+PLG_MEDIA-ACTION_RESIZE_LABEL="Größe ändern"
+PLG_MEDIA-ACTION_RESIZE_PARAM_WIDTH="Breite"
+PLG_MEDIA-ACTION_RESIZE_PARAM_HEIGHT="Höhe"
+PLG_MEDIA-ACTION_RESIZE_QUALITY="Qualität"
+PLG_MEDIA-ACTION_RESIZE_XML_DESCRIPTION="Fügt eine Größe ändern für Bilder hinzu."
```
Comment (tecpromotion) [inline administrator/language/de-DE/plg_media-action_resize.ini]: kein Deutsch

File: administrator/language/de-DE/plg_media-action_resize.sys.ini [hunk 5]
```diff
@@ -0,0 +1,8 @@
+; Joomla! German Translation
+; Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.
+; Copyright (C) Translation 2008 - 2019 J!German (www.jgerman.de)
+; License GNU General Public License version 2 or later; see LICENSE.txt
+; Note : All ini files need to be saved as UTF-8
+
+PLG_MEDIA-ACTION_RESIZE="Medienfunktion - Größe ändern"
+PLG_MEDIA-ACTION_RESIZE_XML_DESCRIPTION="Fügt eine Größe ändern für Bilder hinzu."
```
Comment (tecpromotion) [inline administrator/language/de-DE/plg_media-action_resize.sys.ini]: kein Deutsch

File: administrator/language/de-DE/plg_media-action_rotate.ini [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+; Copyright (C) 2005 - 2019 Open Source Matters. All rights reserved.
+; Copyright (C) Translation 2008 - 2019 J!German (www.jgerman.de)
+; License GNU General Public License version 2 or later; see LICENSE.txt
+; Note : All ini files need to be saved as UTF-8
+
+PLG_MEDIA-ACTION_ROTATE="Medienfunktion - Drehen"
+PLG_MEDIA-ACTION_ROTATE_LABEL="Drehen"
+PLG_MEDIA-ACTION_ROTATE_PARAM_ANGLE="Winkel"
+PLG_MEDIA-ACTION_ROTATE_PARAM_DIRECTION="Richtung"
+PLG_MEDIA-ACTION_ROTATE_PARAM_DIRECTION_RIGHT="Recht"
```
Comment (tecpromotion) [inline administrator/language/de-DE/plg_media-action_rotate.ini]:
```suggestion
PLG_MEDIA-ACTION_ROTATE_PARAM_DIRECTION_RIGHT="Rechts"
```

File: administrator/language/de-DE/plg_media-action_resize.sys.ini (hunk 5 above)
Comment (zero-24) [reply administrator/language/de-DE/plg_media-action_resize.sys.ini]:
```suggestion
PLG_MEDIA-ACTION_RESIZE_XML_DESCRIPTION="Fügt eine „Größe ändern“-Funktion für Bilder hinzu."
```

?

File: administrator/language/de-DE/plg_media-action_resize.ini (hunk 4 above)
Comment (zero-24) [reply administrator/language/de-DE/plg_media-action_resize.ini]:
```suggestion
PLG_MEDIA-ACTION_RESIZE_XML_DESCRIPTION="Fügt eine „Größe ändern“-Funktion für Bilder hinzu."
```

?

**Pull request merged**